### PR TITLE
1.2.0 is here!!!

### PR DESCRIPTION
**Pull request by NotRealPaz**
+yurinet chat to the chat-log
+request node engine 8.6.0
-remove word detection

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -13,124 +13,137 @@ let config = require('./config.json');
...
-	console.log(e.stack)
+  console.log(e.stack)
 }
 
-let detectword = ['กระตุก', 'แลค', 'จอดำ', 'ออนไลน์', 'ค้าง', 'DDwrapper'];
-let DDwrapper = ['กระตุก', 'แลค', 'ค้าง', 'DDwrapper', 'เมนู']
-let yurinet = ['yurinet', 'ออนไลน์', 'online']
-let game = ['game', 'ตัวเกม', 'เกม', 'yuri']
-let black_screen = ['จอดำ', 'black_screen']
 client.on('message', async message => {
```
Comment (icharge) [inline index.js:27]: **client.on** must followed after **client.login** that inside **try..catch** block

File: index.js [hunk 2]
```diff
@@ -13,124 +13,137 @@ let config = require('./config.json');
...
+  let chatminutes = (tempchatminutes < 10) ? `0${tempchatminutes}` :
+    tempchatminutes
+  let chatseconds = (tempchatseconds < 10) ? `0${tempchatseconds}` :
+    tempchatseconds
 
-// Now look at this net
-function net() { // that I just found!
-	// When I say go,
-	// be ready to throw!
+  function getchatdaze() {
```
Comment (icharge) [inline index.js]: **getchatdaze()** must be **getChatDate()**
and **getchatintime()** -> **getChatTime()**

You can merge this two method into one method instead and pass Date parameter to.

File: index.js [hunk 3]
```diff
@@ -13,124 +13,137 @@ let config = require('./config.json');
...
-		.then(link => {
-			console.log(`invite link: ${link}`);
-		});
-}
+  function getchatmsg() {
+    return '``' + response.name + '``' + " : " + response.message
+  }
 
-function DDwrapperMessage() {
+  function getchatformatdiscord() {
```
Comment (icharge) [inline index.js]: **getchatformatdiscord()** -> **formatChatMessage()**

File: index.js (hunk 1 above)
Comment (NotRealPaz) [reply index.js:27]: so put that try catch block on bottom line?

File: index.js [hunk 4]
```diff
@@ -13,124 +13,137 @@ let config = require('./config.json');
...
+    return '``' + getchatdaze() + " " + getchatintime() + "`` " +
+      getchatmsg()
+  }
 
-}
+  function getchatformat() {
```
Comment (icharge) [inline index.js]: **printDiscordChatMessage()**

**Review by icharge — CHANGES_REQUESTED** [review]
You need to name parameters and methods in Naming Convention too.

**Review by NotRealPaz — COMMENTED** [review]
Quick Fix

File: index.js (hunk 1 above)
Comment (icharge) [reply index.js:27]: No, Just ignore this comment. :P

File: index.js [hunk 5]
```diff
@@ -109,25 +103,25 @@ socket.on('lobby.chat', function(response) {
   let chatseconds = (tempchatseconds < 10) ? `0${tempchatseconds}` :
```
Comment (icharge) [inline index.js]: chatSeconds, tempChatSeconds

File: index.js [hunk 6]
```diff
@@ -109,25 +103,25 @@ socket.on('lobby.chat', function(response) {
...
     return chathours + ":" + chatminutes + ":" + chatseconds
   }
 
   function getchatmsg() {
     return '``' + response.name + '``' + " : " + response.message
   }
 
-  function getchatformatdiscord() {
-    return '``' + getchatdaze() + " " + getchatintime() + "`` " +
+  function formatChatMessage() {
```
Comment (icharge) [inline index.js]: merge **getChatDate** and **getChatTime** into **formatChatMessage**

**Review by icharge — CHANGES_REQUESTED** [review]
fix please

**Review by icharge — APPROVED** [review]

**Pull request merged**